Project-Uni/RootShare
# Message Notifications

Shows snackbar notifications upon receiving new messages .
(We'll create a whole message system at some point after migrating the message socket to a separate server where it shows read and unread and all that good stuff)

Testing:
 - Send message to your other account while it's on the platform
 - Send message when the other account is in the conversation to make sure the snackbar doesn't show up then
 - Back out of the conversation and move around to make sure snackbar shows up during appropriate times
